vue/components/ui/molecules/form-input/form-input.vue

```diff
@@ -168,6 +180,7 @@ export const FormInput = {
         headerClasses() {
             const base = {};
             if (this.headerVariant) base[`${this.headerVariant}`] = true;
+            if (this.tooltipProps) base.tooltip = true;

```

Comment:
This shouldn't be needed, tooltip already has a class so if the idea is to have a class to select the header when using tooltip, you can do a selector like: `form-input .tooltip > .header`

Reply:
That's happening because of what we talked about now, tooltip can't be always in use or will break compatibility